csharp/src/Ice/Endpoint.cs

```diff
@@ -59,11 +59,10 @@ public abstract class Endpoint : IEquatable<Endpoint>
         /// <summary>The timeout for the endpoint in milliseconds. 0 means non-blocking, -1 means no timeout.</summary>
         public abstract int Timeout { get; }
 
-        /// <summary>The EndpointType of this endpoint.</summary>
+        /// <summary>The <see cref="EndpointType">type</see> of this endpoint.</summary>
         public abstract EndpointType Type { get; }
-
-        /// <summary>The underlying endpoint.</summary>
-        public virtual Endpoint? Underlying => null;
+        /// <summary>The endpoint's transport unique name.</summary>
+        public abstract string Transport { get; }
```

Comment:
Should we provide a default implementation for this property?
`=> Type.ToString().ToLowerInvariant()`

Reply:
Actually we have Name that does the same, maybe we don't need `Transport`